packages/client/src/client/components/Button.tsx

```diff
@@ -52,8 +53,12 @@ const primaryStyle = css`
 `
 const outlineStyle = css`
   ${({ theme }) => `
-  color: ${theme.newTheme.color["Component colors/Components/Buttons/Brand/button-brand-bg"]};
+  color: ${theme.newTheme.color["Colors/Text/text-brand-primary (900)"]};
   border: 1px solid ${theme.newTheme.color["Colors/Border/border-brand"]};
+
+  &:hover {
+    opacity: 0.65;
```

Comment:
you know what I'm going to say 😉 
seriously, we had better get an update from UX on this - see if getting rid of opacity is still the strategy with our UX design systems